projects/fsastorefrontstyles/scss/components/account/_premium-calendar.scss

```diff
@@ -65,6 +65,16 @@
         }
       }
     }
+    .icon-triangle-bottom {
+      position: absolute;
+      right: 5%;
+      top: 5%;
+      @include fs-transition;
+      &.rotate {
+        transform: translateY(-5%) rotateZ(180deg);
+        -webkit-transform: translateY(-5%) rotateZ(180deg);
+      }
+    }
```

Comment:
You already have very similar if not the same styling for notice class in this same file. Can this be refactored in one place?